js/sign/src/cli.ts

```diff
@@ -24,12 +28,17 @@ function readOptions() {
 export function main() {
   const options = readOptions();
   const webBundle = fs.readFileSync(options.input);
-
+  const parsedPrivateKey = parsePemKey(
+    fs.readFileSync(options.privateKey, 'utf-8')
+  );
   const signer = new IntegrityBlockSigner(webBundle, {
-    key: parsePemKey(fs.readFileSync(options.privateKey, 'utf-8')),
+    key: parsedPrivateKey,
   });
   const integrityBlock = signer.sign();
 
+  const consoleLogColor = '\x1b[32m'; // Green
+  console.log(`${consoleLogColor}${new WebBundleId(parsedPrivateKey)}`);
+
```

Comment:
You should reset the console color with `'\x1b[0m'`, otherwise all subsequent output will be green :)